r-lib-error-1.0.md

```diff
@@ -0,0 +1,64 @@
+
+# Specification of r-lib errors, version 1.0.0
+
+Note: this is an experimental draft.
+
+Note: version 1.0.0 corresponds to the `rlang_error` class, as
+implemented in rlang 0.4.0.
+
+## Classes
+
+An r-lib error object always has at least classes `rlib_error`, `error`
+and `condition`. It may have other classes.
+
+## Members
```

Comment:
Should we call these fields?

It might also be worth reminding the reader that a condition is a list and these are all named elements of that list.

Reply:
OK.